metabase/metabase
# Slack app settings

Epic https://github.com/metabase/metabase/issues/19435

Adds UI to configure slack app.

<img width="1728" alt="Screenshot 2022-01-10 at 14 37 52" src="https://user-images.githubusercontent.com/8542534/148760463-4bd04bce-3576-4db5-9835-42c382622e30.png">

How to test:
- Merge https://github.com/metabase/metabase/pull/19592 locally into this branch
- Make sure it is possible to create a slack app following the instructions in Admin > Slack
